accepted/future-releases/static-extension-methods/feature-specification.md

```diff
@@ -673,6 +627,18 @@ The use of `typedef` for something which is not a type may be too confusing. Ano
 extension MyWidgetList<T extends Widget> = prefix.MyList<T>;
 ```
 
+### Avoiding Scope Conflicts
+
+We allow importing two extensions with the same name into the same scope, and both of them *work*. They cannot be referenced by name, but they can be implicitly invoked. This breaks the current invariant that we cannot have two declarations with the same name in the same scope.
+
+To retain this invariant, we can define a "fresh extension alias" for each extension declaration. It is a second *fresh name* for the extension (a name not occurring anywhere else in the program), which will be a fresh *private* name if the extension's name is private (or omitted, in which case we only need the one fresh name), and a fresh non-private name if the extension's name is non-private. Any implicit extension invocation uses the *fresh extension alias* name to refer to the extension (so an `expr.foo()` which resolves to a method on the extension  `Ext` is equivalent to `$fresh$Ext(expr).foo()`). 
```

Comment:
If you do use fresh names, best to make them private.  They shouldn't be exported.  See below.

Reply:
The goal is to make them exported, so they can be used to reference the extension even when the declared name is not in scope.